src/components/calcite-shell-center-row/calcite-shell-center-row.scss

```diff
@@ -0,0 +1,58 @@
+:host {
+  @extend %component-host;
+  transition: height var(--calcite-app-animation-time) ease-in-out;
+
+  --calcite-app-shell-center-row-height-small: 33%;
+  --calcite-app-shell-center-row-height-medium: 50%;
+  --calcite-app-shell-center-row-height-large: 100%;
+}
+
+@import "../../scss/includes/_component";
+
+.content {
+  background-color: var(--calcite-app-background-content);
+  border: 1px solid var(--calcite-app-border);
+  flex: 1 0 0;
+  height: 100%;
+  margin: 0;
+  overflow: hidden;
+  width: 100%;
+}
+
+:host([detached]) {
+  animation: calcite-app-fade-in-up var(--calcite-app-animation-time) var(--calcite-app-easing-function);
+  border-radius: var(--calcite-app-border-radius);
+  box-shadow: var(--calcite-app-shadow-0);
+  margin: var(--calcite-app-cap-spacing-half) var(--calcite-app-side-spacing-half)
+    var(--calcite-app-cap-spacing-plus-half);
+  overflow: hidden;
+}
+
+:host([position="end"]) {
```

Comment:
I don't think we need the position do we? Since its always centered?

Reply:
Ah ha! This is for top and bottom alignment.

`end`
![image](https://user-images.githubusercontent.com/12503298/80813677-54eac180-8b7f-11ea-8f96-da5138cad27f.png)
 `start`
![image](https://user-images.githubusercontent.com/12503298/80813705-6338dd80-8b7f-11ea-8bff-3819b786d3ff.png)

Reply:
oh nice! 👍